addons/sms/static/src/js/fields_sms_widget.js

```diff
@@ -23,6 +24,15 @@ var SmsWidget = FieldText.extend({
         this.nbrChar = 0;
         this.nbrSMS = 0;
         this.encoding = 'GSM7';
+        this.enable_emojis = !!this.nodeOptions.enable_emojis;
+    },
+    /**
+     * This will add the emoji dropdown to a target field (controlled by the "enable_emojis" attribute).
+     * @override
+     */
+    _attachEmojisDropdown: function () {
+        if(this.enable_emojis)
```

Comment:
Linting:
- Put a space after the "if"
  If you linter is correctly configured, it should warn you:
![image](https://user-images.githubusercontent.com/41282536/80070054-a4f3d380-8542-11ea-9ec5-906a7493ff54.png)
- Put brackets {} between the statement, even if only one line of code
```
if (this.enableEmojis) {
    fieldAttachEmojis.apply(this, arguments);
}
```